build/CMakeCache.txt

```diff
@@ -0,0 +1,360 @@
+# This is the CMakeCache file.
+# For build in directory: /home/ksc/work/ser4/solar_car_control_system/build
+# It was generated by CMake: /usr/bin/cmake
+# You can edit this file to change values found and used by cmake.
+# If you do not want to change any of the values, simply exit the editor.
+# If you do want to change a value, simply edit, save, and exit the editor.
+# The syntax for the file is as follows:
+# KEY:TYPE=VALUE
+# KEY is the name of a variable in the cache.
+# TYPE is a hint to GUIs for the type of VALUE, DO NOT EDIT TYPE!.
+# VALUE is the current value for the KEY.
+
+########################
+# EXTERNAL cache entries
+########################
+
+//Path to a program.
+CMAKE_ADDR2LINE:FILEPATH=/usr/bin/addr2line
+
+//Path to a program.
+CMAKE_AR:FILEPATH=/usr/bin/ar
+
+//No help, variable specified on the command line.
+CMAKE_BUILD_TYPE:STRING=Release
```

Comment:
Is this file in use? I guess not? -> remove

Reply:
Do we need anything from build in git repo?
Comes in by accident.

Reply:
I do not think that we need anything. Might be a directory that we should put into the `.gitignore` file as well.

Reply:
ignored by .gitignore